package.json

```diff
@@ -56,12 +56,12 @@
   ],
   "theiaPluginsDir": "plugins",
   "theiaPlugins": {
-    "vscode-builtin-cpp": "https://open-vsx.org/api/vscode/cpp/1.43.2/file/vscode.cpp-1.43.2.vsix",
-    "vscode-builtin-json": "https://open-vsx.org/api/vscode/json/1.39.1/file/vscode.json-1.39.1.vsix",
-    "vscode-builtin-markdown": "https://open-vsx.org/api/vscode/markdown/1.39.1/file/vscode.markdown-1.39.1.vsix",
-    "vscode-builtin-npm": "https://open-vsx.org/api/vscode/npm/1.39.1/file/vscode.npm-1.39.1.vsix",
-    "vscode-builtin-typescript": "https://open-vsx.org/api/vscode/typescript/1.39.1/file/vscode.typescript-1.39.1.vsix",
-    "vscode-builtin-typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.39.1/file/vscode.typescript-language-features-1.39.1.vsix",
-    "vscode-clangd": "https://open-vsx.org/api/llvm-vs-code-extensions/vscode-clangd/0.1.5/file/llvm-vs-code-extensions.vscode-clangd-0.1.5.vsix"
+    "vscode-builtin-cpp": "https://open-vsx.org/api/vscode/cpp/1.52.1/file/vscode.cpp-1.52.1.vsix",
```

Comment:
`1.52.1` might be a bit too recent- even the latest  Theia framework does not completely support this version of the vscode extensions API, I believe. 

Maybe the easiest is to align with the versions in the main example application, for those extensions we use here? 

https://github.com/eclipse-theia/theia/blob/master/package.json#L86-L151

Reply:
I agree, it's simple to just align with this. But how can we avoid this kind of behaviour ? If I undersant corretly, according to [this](https://blogs.eclipse.org/post/brian-king/new-era-open-vsx-registry) it's an exceptional event which happend here, but it's not easy to find which version to use when the default version doesn't exist anymore since open-vsx show you the latest version available.
It could be worth to write somewhere which version we use and why we use this version. And maybe it's preferable to use the same version for all the built in extension ?
It could be worth to write somewhere what version we use and why we use this version.

Reply:
@irbyk generally we choose a version which satisfies the framework’s default API version (compatibility is determined but not guaranteed through an extension’s `engines.vscode` field). If we notice an extension is misbehaving (such as markdown-language-features using an unsupported API) we downgrade the extension to a version which works.

Reply:
I understand (Sorry, I use another account for my previous message but I hadn't notice until now). Do I modify the version to have the same number as the main example ?

Reply:
Yes, I think that would be best for now. FYI recent improvements may soon make unnecessary, such manual management of the exact version of vscode-builtins we pull.